system/draft/TestAuditSuite.py

```diff
@@ -5,55 +5,64 @@
 import logging
 logger = logging.getLogger(__name__)
 
+
 @pytest.mark.usefixtures('docker_setup_and_teardown')
 class TestAuditSuite:
 
     @pytest.mark.asyncio
-    async def test_case_restart_one_node(self, pool_handler, wallet_handler, get_default_trustee):
+    async def test_case_restart_one_node(
+            self, pool_handler, wallet_handler, get_default_trustee, nodes_num
+    ):
         trustee_did, _ = get_default_trustee
         test_nodes = [NodeHost(i) for i in range(1, 8)]
+        # ensure?
```

Comment:
Why this question is here (and similar questions lower)? What possible improvement is implied?

Reply:
I've replaced common nyms sending with new ensure helpers. Test run time remains pretty the same.